swan-lake/featured-scenarios/manage-data-persistence-with-bal-persist.md

```diff
@@ -143,7 +143,7 @@ The `persist generate` will parse the `persist/model.bal` definition file and ge
 
 > Note: All of the above auto-generated BAL files should not be modified.
 
-> Note: If you use other data stores like MySQL or Google Sheets, additional files are created to handle runtime configurations and scripts to set up the database/worksheet. Since `in-memory` is used, you don’t need any configuration and pre setting up to run the application.
+> Note: If you use other data stores like MySQL, MSSQL or Google Sheets, additional files are created to handle runtime configurations and scripts to set up the database/worksheet. Since `in-memory` is used, you don’t need any configuration and pre setting up to run the application.
```

Comment:
```suggestion
> Note: If you use other data stores like MySQL, MSSQL, or Google Sheets, additional files will get created to handle runtime configurations and scripts to set up the database/worksheet. Since `in-memory` is used, you don’t need to do any configuration or setting-up before running the application.
```